### PR TITLE
Myrackspace

### PR DESCRIPTION
**Pull request by Sterling463**

### CODE REVIEW

File: content/dedicated-hosting/reset-user-password-on-active-directory-domain.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+permalink: reset-user-password-on-active-directory-domain/
+audit_date:
+title: Reset User Password on Active Directory Domain
```
Comment (catlook) [inline content/dedicated-hosting/reset-user-password-on-active-directory-domain.md]: title: Reset user password on Active Directory domain

**Comment by Sterling463** [comment]
@stephamon do you mind reviewing this?

**Comment by rackernexus** [comment]
Build 930 was successful. Staging content [Preview!](https://support-revs.staging.deconst.org/build-28f09ec4c/how-to/). valid for 14 days

**Comment by rackernexus** [comment]
Build 931 was successful. Staging content [Preview!](https://support-revs.staging.deconst.org/build-b0a65c039/how-to/). valid for 14 days

**Comment by rackernexus** [comment]
Build 936 was successful. Staging content [Preview!](https://support-revs.staging.deconst.org/build-661c7b20a/how-to/). valid for 14 days

**Comment by rackernexus** [comment]
Build 937 was successful. Staging content [Preview!](https://support-revs.staging.deconst.org/build-8ffbcd63f/how-to/). valid for 14 days

**Comment by stephamon** [comment]
@Sterling463 edits applied!

**Pull request merged**